cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorTest.java

```diff
@@ -1304,36 +1295,36 @@ public ReconciliationState createReconciliationState(Reconciliation r, Kafka ka)
             })));
     }
 
+    @SuppressWarnings("unchecked")
     @ParameterizedTest
     @MethodSource("data")
     @Timeout(value = 2, timeUnit = TimeUnit.MINUTES)
-    public void testReconcile(Params params, VertxTestContext context) {
+    public void testReconcileAllNamespaces(Params params, Vertx vertx, VertxTestContext context) {
```

Comment:
Why are you swapping `testReconcile` and `testReconcileAllNamespaces`? Just to make reviewers life a hell? Not doing it would make it much more clear that you didn't changed any logic of the tests.

Reply:
I think this is a change I rebased in, will double check.

Reply:
Okay, looks I made that change somehow, that's indeed odd. Fixing it up.

Reply:
PS, I'm actively pursuing a strategy of not making reviewers' lives hell ;) 

So, cheers for catching this :)